### ISSUE TITLE
Sort Spoon report by order of execution

### ISSUE THREAD

**Issue by miraculix-druids** [square/spoon#365]
Currently, the test classes in the Spoon report is sorted alphabetically. I would like to see the report exactly in the order that it was executed.

**Comment by JakeWharton** [comment]
What happens when different devices execute them in different orders?

**Comment by miraculix-druids** [comment]
A test Suite with @Suite.SuiteClasses({...}) is always executed in the same order.

**Comment by JakeWharton** [comment]
What about all the other ways of running?

**Comment by miraculix-druids** [comment]
Fair enough. The best solution would be to make the order configurable with a flag.

**Comment by tir38** [comment]
I'm curious why you want this? What benefit do you have in seeing the order your tests were run in that running your tests in Android Studio doesn't give you?

**Comment by Naitbit** [comment]
I once had a test that did not clean after itself properly, and caused a failure of the next test. Finding in what order tests were executed would make finding the issue easier in that case. However I do not know how often people have such an issues, I had it only once and I've fixed it without having ordered spoon report, so that may not be good enough use case.
